### ISSUE TITLE
Player not notified on /unmute or when /tempmute ends

### ISSUE THREAD

**Issue by acidbyt3** [BanManagement/BanManager#602]
Title says all.
It's a very handy (and basic) feature, yet I can't even find anything on messages.yml that would suggest this feature is implemented.

Am I missing something?

**Comment by confuser** [comment]
Good question.

/unmute is definitely missing a notification if the player is online, that can be added.

As for when temp mutes end, that's more expected due to the way it's implemented. 

When a player sends a message, it checks if they have any mutes, it also does the expiration check there; but lets the message go through without any unmuted notification. In addition, the mute syncing (which enables global ban/muting) also performs this check. 

tl;dr The notification to the player will be delayed

**Comment by acidbyt3** [comment]
Okay yeah, that makes sense then, not getting a notification once the mute expires.
Tying the notification event to the global sync would probably be too much hassle/overcomplicated and not ideal seeing as the global sync only triggers every couple minutes. The notification would most likely come through late.

However a manual /unmute notification to the receiving user would probably be a nice addition, I think. :)
